CsharpExtras/Extensions/IEnumerableExtension.cs

```diff
@@ -36,21 +40,29 @@ public static class IEnumerableExtension
             }
         }
 
-        public static int FirstIndexOf<T>(this IEnumerable<T> values, T value)
+        public static int FirstIndexOfOneBased<T>(this IEnumerable<T> values, T value) =>
+            values.FirstIndexOf(value, 1);
+        public static int FirstIndexOfZeroBased<T>(this IEnumerable<T> values, T value) =>
+            values.FirstIndexOf(value, 0);
+
+        public static int FirstIndexOf<T>(this IEnumerable<T> values, T value, int startIndex)

```

Comment:
The name `startIndex` is a bit confusing in this context. At first glace I thought it was the index where the search starts (i.e. if you say the start index is 5, it will start looking for a match at position 5 of the array).